### PR TITLE
Release note v10.9.0

### PR DESCRIPTION
**Pull request by albertotessarotto**
## Pull Request Type

- [ ] Bugfix
- [ ] Documentation content changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensible content has been committed

### CODE REVIEW

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+### New Marketplace Components
+
+### Marketplace Updates
+
+## Backoffice
+
+## How to update your Console
+
+For on-premise Console installations, please contact your Mia Platform referent to know how to use the `Helm chart version X.X.X`.
```
Comment (ugho16) [inline docs/release-notes/platform_10-9-0_releasenotes.md:46]:
```suggestion
For on-premise Console installations, please contact your Mia-Platform referent to know how to use the `Helm chart version X.X.X`.
```

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+## Console
+
+### New Features
+
+#### Managing Service Accounts at Console level
+
+It is possible to create [Company-independent service accounts](TO-DO link) in case you need to give them privileges to perform [root-level operations](/development_suite/identity-and-access-management/console-levels-and-permission-management.md).
+
+:::info
+Only Console Super Users can [add](TO-DO link) or [delete](TO-DO link) Company independent service accounts, manageable exclusively at back-office level.
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:19]:
```suggestion
Only Console Super Users can [add](TO-DO link) or [delete](TO-DO link) Company independent service accounts, manageable exclusively at backoffice level.
```

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+:::
+
+### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+*
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:26]:
```suggestion
* We have solved a bug in the creation of a microservice that prevented the insertion of a Git repository owner with an underscore in its name
```

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+It is possible to create [Company-independent service accounts](TO-DO link) in case you need to give them privileges to perform [root-level operations](/development_suite/identity-and-access-management/console-levels-and-permission-management.md).
+
+:::info
+Only Console Super Users can [add](TO-DO link) or [delete](TO-DO link) Company independent service accounts, manageable exclusively at back-office level.
+:::
+
+### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:24]: if there's just one bug, I'd rephrase it

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+* 
+
+## Fast Data
+
+### New Features
+
+Support to .sql file type in the DDL modal import
```
Comment (TheRusCompass) [inline docs/release-notes/platform_10-9-0_releasenotes.md:32]: @giuliowaitforitdavide let's rephrase it better, giving a more exhaustive explanation

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+### Improvements
+
+One to one relationships in aggregation as object instead of array.
```
Comment (TheRusCompass) [inline docs/release-notes/platform_10-9-0_releasenotes.md:36]: @giuliowaitforitdavide let's rephrase it better, giving a more exhaustive explanation

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+However, as with any major update, there are a few breaking changes to be aware of. We've renamed the `STRATEGIES_FOLDER` environment variable to `MANUAL_STRATEGIES_FOLDER` and the `CUSTOM_FUNCTIONS_FOLDER` environment variable to `TRIGGER_CUSTOM_FUNCTIONS_FOLDER`. We've also removed the `USE_AUTOMATIC` variable. Additionally, the `SINGLE_VIEW_NAME`,` ER_SCHEMA_FOLDER`, and `PROJECTION_CHANGES_SCHEMA_FOLDER` variables are now required. Check out the official documentation (TODO: add link)!
+
+Do not forget to check out the [changelog](/runtime-suite/single-view-trigger-generator/changelog.md) for a full list of changes!
+
+### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* Projections and SoR for the no-code ERSchema still shown even if deleted
```
Comment (TheRusCompass) [inline docs/release-notes/platform_10-9-0_releasenotes.md:51]: @giuliowaitforitdavide let's change the perspective here, writing what is possible to be done now without describing what was not possible before

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+* Single View Creator that could be attached to more Single Views
```
Comment (TheRusCompass) [inline docs/release-notes/platform_10-9-0_releasenotes.md:52]: @giuliowaitforitdavide let's change the perspective here, writing what is possible to be done now without describing what was not possible before

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+One to one relationships in aggregation as object instead of array.
+
+### Breaking Changes
+
+#### Revamped Single View Trigger Generator
+We're thrilled to announce the release of v.2.0.1 of the Single View Trigger Generator, one of the Fast Data component. In this release, we've given the component a complete overhaul to optimize its performance and streamline its functionality.
+
+However, as with any major update, there are a few breaking changes to be aware of. We've renamed the `STRATEGIES_FOLDER` environment variable to `MANUAL_STRATEGIES_FOLDER` and the `CUSTOM_FUNCTIONS_FOLDER` environment variable to `TRIGGER_CUSTOM_FUNCTIONS_FOLDER`. We've also removed the `USE_AUTOMATIC` variable. Additionally, the `SINGLE_VIEW_NAME`,` ER_SCHEMA_FOLDER`, and `PROJECTION_CHANGES_SCHEMA_FOLDER` variables are now required. Check out the official documentation (TODO: add link)!
+
+Do not forget to check out the [changelog](/runtime-suite/single-view-trigger-generator/changelog.md) for a full list of changes!
```
Comment (danibix95) [inline docs/release-notes/platform_10-9-0_releasenotes.md:45]: The changelog is going to not be available anymore. Let's remove this sentence

File: docs/release-notes/platform_10-9-0_releasenotes.md (hunk 5 above)
Comment (giuliowaitforitdavide) [reply docs/release-notes/platform_10-9-0_releasenotes.md]: Changed

File: docs/release-notes/platform_10-9-0_releasenotes.md (hunk 6 above)
Comment (giuliowaitforitdavide) [reply docs/release-notes/platform_10-9-0_releasenotes.md]: Changed

File: docs/release-notes/platform_10-9-0_releasenotes.md (hunk 7 above)
Comment (giuliowaitforitdavide) [reply docs/release-notes/platform_10-9-0_releasenotes.md]: Changed

File: docs/release-notes/platform_10-9-0_releasenotes.md (hunk 8 above)
Comment (giuliowaitforitdavide) [reply docs/release-notes/platform_10-9-0_releasenotes.md]: Changed

File: docs/release-notes/platform_10-9-0_releasenotes.md (hunk 9 above)
Comment (giuliowaitforitdavide) [reply docs/release-notes/platform_10-9-0_releasenotes.md:45]: Btw if we remove the CHANGELOD could we add that these changes refer to the single view trigger generator? I was confused from this as I thought these release notes impact my review of the fast data docs

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+image: "img/release-note-link-preview.png"
+---
+
+_May 04, 2023_
+
+## Console
+
+### New Features
+
+#### Selection of default providers for the Company's projects
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:14]:
```suggestion
#### Selection of default Providers for the Company's Projects
```

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+
+Within the Providers section in the Company Overview area it is now possible to find a new feature that allows the user with Company Owner permissions to be able to set the Default providers that will use all the new projects created on the respective company.
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:16]:
```suggestion
In the Company Overview area, inside the Providers section, in it is now possible to find a new feature that allows the user with Company Owner permissions to be able to set the Default Providers that will be used in all the new Projects created in the respective Company.
```

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+
+![Default Providers Selection](./img/v10.9_defaultProviders.png)
+
+This will enhance the governance for every Company Owner for company's projects and give better control over the management of company tools.
+The provider for each capability (Git Provider, CI/CD and Secret Manager) can always be changed in any moment via CMS.
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:21]:
```suggestion
This will enhance the governance of Company Owners on all the Projects of their Companies, granting them better control over the management of company tools.
The Provider for each capability (Git Provider, CI/CD and Secret Manager) can always be changed at any moment via CMS.
```

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+
+For more information, [go to the documentation](/development_suite/set-up-infrastructure/configure-provider.mdx)
+
+#### GitHub Actions as CI/CD tool in Console
+
+Starting with this version, the console supports GitHub Actions as a CI/CD tool.
+
+Within the Providers section accessible through the Company Overview area you can now configure a GitHub provider as a CI/CD tool.
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:29]:
```suggestion
Starting with this version, Mia-Platform Console supports GitHub Actions as a CI/CD tool.

Within the Providers section, accessible through the Company Overview area, you can now configure a GitHub provider as a CI/CD tool.
```

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+
+## Fast Data
+
+### New Features
+
+The file picker of the DDL modal import now support `.sql` file 
+
+### Improvements
+
+The console automatic aggregation generation now represents one to one relationship configuration as object.
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:55]:
```suggestion
The Console automatic aggregation generation now represents one-to-one relationship configuration as an object.
```

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+
+However, as with any major update, there are a few breaking changes to be aware of. We've renamed the `STRATEGIES_FOLDER` environment variable to `MANUAL_STRATEGIES_FOLDER` and the `CUSTOM_FUNCTIONS_FOLDER` environment variable to `TRIGGER_CUSTOM_FUNCTIONS_FOLDER`. We've also removed the `USE_AUTOMATIC` variable. Additionally, the `SINGLE_VIEW_NAME`,` ER_SCHEMA_FOLDER`, and `PROJECTION_CHANGES_SCHEMA_FOLDER` variables are now required. Check out the official documentation (TODO: add link)!
+
+Do not forget to check out the [changelog](/runtime-suite/single-view-trigger-generator/changelog.md) for a full list of changes!
+
+### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* The Projections and the SoR are now deleted from the No-Code ERSchema if deleted from their console sections
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:70]:
```suggestion
* The Projections and the SoR are now deleted from the No-Code ERSchema if deleted from their Console sections
```

File: docs/release-notes/platform_10-9-0_releasenotes.md (hunk 9 above)
Comment (davide-sera) [reply docs/release-notes/platform_10-9-0_releasenotes.md]: I removed the changelog line :+1:

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 16]
```diff
@@ -0,0 +1,83 @@
...
+### Improvements
+
+One to one relationships in aggregation as object instead of array.
+
+### Breaking Changes
+
+#### Revamped Single View Trigger Generator
+We're thrilled to announce the release of v.2.0.1 of the Single View Trigger Generator, one of the Fast Data component. In this release, we've given the component a complete overhaul to optimize its performance and streamline its functionality.
+
+However, as with any major update, there are a few breaking changes to be aware of. We've renamed the `STRATEGIES_FOLDER` environment variable to `MANUAL_STRATEGIES_FOLDER` and the `CUSTOM_FUNCTIONS_FOLDER` environment variable to `TRIGGER_CUSTOM_FUNCTIONS_FOLDER`. We've also removed the `USE_AUTOMATIC` variable. Additionally, the `SINGLE_VIEW_NAME`,` ER_SCHEMA_FOLDER`, and `PROJECTION_CHANGES_SCHEMA_FOLDER` variables are now required. Check out the official documentation (TODO: add link)!
```
[the remark this answers is not in the document]
Comment (davide-sera) [reply docs/release-notes/platform_10-9-0_releasenotes.md:62]: Ok for me. I'll wait for @TheRusCompass then

File: docs/release-notes/img/v10.9_defaultProviders.png
Comment (marcofilippi) [inline docs/release-notes/img/v10.9_defaultProviders.png:1]: Here the image shows also the "License" section

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 17]
```diff
@@ -0,0 +1,124 @@
...
+
+For more information, [go to the documentation](/development_suite/set-up-infrastructure/configure-provider.mdx).
+
+#### GitHub Actions as CI/CD tool in Console
+
+Starting with this version, the console supports GitHub Actions as a CI/CD tool.
+
+Within the Providers section accessible through the Company Overview area you can now configure a GitHub provider as a CI/CD tool and using it for the deployment of your workload.
+
+For more information, [go to the documentation](/development_suite/deploy/.mdx).
```
Comment (davide-sera) [inline docs/release-notes/platform_10-9-0_releasenotes.md:31]:
```suggestion
For more information, [go to the documentation](/development_suite/deploy/deploy.mdx).
```
This link is broken. Did we mean the suggested one?
image

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 18]
```diff
@@ -0,0 +1,124 @@
...
+This will enhance the governance for every Company Owner for company's projects and give better control over the management of company tools.
+The provider for each capability (Git Provider, CI/CD and Secret Manager) can always be changed in any moment via CMS.
+
+For more information, [go to the documentation](/development_suite/set-up-infrastructure/configure-provider.mdx).
+
+#### GitHub Actions as CI/CD tool in Console
+
+Starting with this version, the console supports GitHub Actions as a CI/CD tool.
+
+Within the Providers section accessible through the Company Overview area you can now configure a GitHub provider as a CI/CD tool and using it for the deployment of your workload.
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:29]:
```suggestion
Starting with this version, Mia-Platform Console supports GitHub Actions as a CI/CD tool.

Within the Providers section, accessible through the Company Overview area, you can now configure a GitHub provider as a CI/CD tool and use it for the deployment of your workload.
```

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 19]
```diff
@@ -0,0 +1,124 @@
...
+Furthermore, the following MongoDB operators for `PATCH` action were introduced:
+
+- [`$addToSet`](https://www.mongodb.com/docs/manual/reference/operator/update/addToSet/) (v6.4.0)
+- [`$pull`](https://www.mongodb.com/docs/manual/reference/operator/update/pull/) (v6.5.0)
+
+The full [changelog](/runtime_suite/crud-service/changelog.md) can be found in the plugin detail page.
+
+## Backoffice - 1.3.9
+
+### Bug Fix
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:104]: should be removed

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 20]
```diff
@@ -0,0 +1,124 @@
...
+
+#### New component `bk-layout-swap`
+
+New component [`bk-layout-swap`](/business_suite/backoffice/components/misc.md#bk-layout-swap) is available. It performs a layout change listening to `layout/change` event. The payload of the event is the layout rendered by the component. It is bundled separately.
+
+### Bug Fix
+
+#### Form components resolve lookups on creation
+
+Form components correctly resolve lookups in initial values from `add-new` event payload
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:120]:
```suggestion
#### `bk-crud-client` allows to disable url writing

New property `reflectToUrl` allows to control whether `bk-crud-client` reflects its state to the URL with a `window.history.pushState`.

#### New component `bk-layout-swap`

New component [`bk-layout-swap`](/business_suite/backoffice/components/misc.md#bk-layout-swap) is available. It performs a layout change listening to `layout/change` event. The payload of the event is the layout rendered by the component. It is bundled separately.

### Bug Fix

#### Form components resolve lookups on creation

Form components correctly resolve lookups in initial values from `add-new` event payload.
```

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 21]
```diff
@@ -0,0 +1,124 @@
...
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* The Projections and the SoR are now deleted from the No-Code ERSchema if deleted from their Console sections
+* The Single View Creator is not attachable to only one Single View
+* Mongo operators are now properly interpolated in aggregations executed by the Single View Creator. This fix has been addressed in version 5.6.6 of the Single View Creator.
+
+## Marketplace
+
+### New Marketplace Components
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:83]: should be removed if there are no new marketplace components

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 22]
```diff
@@ -0,0 +1,124 @@
...
+However, as with any major update, there are a few breaking changes to be aware of. We've renamed the following environment variables to better clarify their purpose:
+- `STRATEGIES_FOLDER` → `MANUAL_STRATEGIES_FOLDER`
+- `CUSTOM_FUNCTIONS_FOLDER` → `TRIGGER_CUSTOM_FUNCTIONS_FOLDER`.
+We've also removed the `USE_AUTOMATIC` variable since the feature is now enabled in all the service configurations.  
+Furthermore, the following environment variables are now required:
+- `SINGLE_VIEW_NAME`
+- `ER_SCHEMA_FOLDER`
+- `PROJECTION_CHANGES_SCHEMA_FOLDER`
+
+For more details on the service and its behavior, please check out the official [documentation](/fast_data/single_view_trigger_generator.md)
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:71]:
```suggestion
#### Revamped Single View Trigger Generator

We're thrilled to announce the release of v.2.0.1 of the Single View Trigger Generator, one of the Fast Data component. In this release, we've given the component a complete overhaul to optimize its performance and streamline its functionality.

However, as with any major update, there are a few breaking changes to be aware of. We've renamed the following environment variables to better clarify their purpose:
- `STRATEGIES_FOLDER` → `MANUAL_STRATEGIES_FOLDER`
- `CUSTOM_FUNCTIONS_FOLDER` → `TRIGGER_CUSTOM_FUNCTIONS_FOLDER`.
We've also removed the `USE_AUTOMATIC` variable since the feature is now enabled in all the service configurations.  
Furthermore, the following environment variables are now required:
- `SINGLE_VIEW_NAME`
- `ER_SCHEMA_FOLDER`
- `PROJECTION_CHANGES_SCHEMA_FOLDER`

For more details on the service and its behavior, please check out the official [documentation](/fast_data/single_view_trigger_generator.md).
```

File: docs/release-notes/platform_10-9-0_releasenotes.md [hunk 23]
```diff
@@ -0,0 +1,141 @@
...
+
+- `SINGLE_VIEW_NAME`
+- `ER_SCHEMA_FOLDER`
+- `PROJECTION_CHANGES_SCHEMA_FOLDER`
+
+For more details on the service and its behavior, please check out the official [documentation](/fast_data/single_view_trigger_generator.md).
+
+### New Features
+
+The file picker of the Projection DDL modal import, that can be used in the System of Record section, now supports `.sql` file
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-9-0_releasenotes.md:80]:
```suggestion
The file picker of the Projection DDL modal import, that can be used in the System of Record section, now supports `.sql` file. 
```

**Review by fredmaggiowski — APPROVED** [review]
LGTM
